audb/core/publish.py

```diff
@@ -175,6 +175,13 @@ def _find_media(
     def job(file):
         path = os.path.join(db_root, file)
         if file not in deps:
+            # assert lowercase file extensions
+            ext = audeer.file_extension(file)
+            if ext.lower() != ext:
+                raise RuntimeError(
+                    "Only lower case file extensions are allowed, "
```

Comment:
```suggestion
                    "The file extension of media file must be lowercase, "
```

Reply:
Changed to: "The file extension of a media file must be lowercase, "